code/controllers/ProcessScheduler/process.dm

```diff
@@ -303,11 +324,12 @@ datum/controller/process/proc/copyStateFrom(var/datum/controller/process/target)
 datum/controller/process/proc/onKill()
 
 datum/controller/process/proc/onStart()
+	LAGCHECK(world.tick_usage > 100 - tick_allowance)
```

Comment:
This doesn't look right.

Reply:
This is a port from goon so i'll have to ask them

Reply:
@MrStonedOne 
![capture](https://user-images.githubusercontent.com/10489825/46127686-a803d200-c1ee-11e8-89fe-961180952209.PNG)